gulpfile.js

```diff
@@ -225,10 +225,25 @@ gulp.task("replace-js-strings", ["webpack", "minify-js"], function () {
     .pipe(gulp.dest(dirs.dist));
 });
 
+gulp.task("version-check", function () {
+  var shrinkwrapVersion = require("./npm-shrinkwrap")["node-version"];
+  var packageVersion = require("./package").engines.node;
```

Comment:
The package.json is already required here:
https://github.com/mesosphere/marathon-ui/blob/feature/version-check/gulpfile.js#L17